MainWindow.xaml

```diff
@@ -63,6 +63,6 @@
             <ComboBoxItem Content="Portuguese"/>
             <ComboBoxItem Content="Russian"/>
         </ComboBox>
-        <Label Content="Language:" HorizontalAlignment="Left" Margin="364,63,0,0" VerticalAlignment="Top" Width="67"/>
+        <Label Content="Language:" HorizontalAlignment="Left" Margin="385,81,0,0" VerticalAlignment="Top"/>
     </Grid>
 </Window>
```

Comment:
All of the comboboxes have their text not vertically centered, which is resolved by adding `"VerticalContentAlignment="Center"`.